profiles/kentik_snmp/raritan/dominion.yml

```diff
@@ -0,0 +1,66 @@
+# https://mibs.observium.org/mib/RemoteKVMDevice-MIB/
+---
+extends:
+  - if-mib.yml
+  - system-mib.yml
+
+sysobjectid:
+  - 1.3.6.1.4.1.13742.3.2.10    # Raritan Dominion SXII DSX2-32
+  - 1.3.6.1.4.1.13742.*    # Raritan Devices
+
+provider: kentik-switch
+
+metrics:
+  - MIB: RemoteKVMDevice-MIB
+    table:
+      name: remoteKVMDeviceGet
+      OID: 1.3.6.1.4.1.13742.3.1
+    symbols:
+      # Percent current memory usage
+      - name: systemUsageMemory
+        OID: 1.3.6.1.4.1.13742.3.1.1
```

Comment:
Since this device already has an OID for memory in % we should add the below syntax, makes it easier when we do entity dashboards and golden metrics
tag: MemoryUtilization
poll_time_sec: 60

Reply:
Added
      - name: systemUsageMemory
        OID: 1.3.6.1.4.1.13742.3.1.1.0
        tag: MemoryUtilization
        poll_time_sec: 60